lib/elasticsearch.ts

```diff
@@ -70,14 +78,108 @@ function filter_old_indices(current_indices) {
 // was deleted or an error
 function delete_index(index) {
   return new Promise((resolve, reject) => {
-    request_es("del", `/${index}`).then(() => resolve(index)).catch(reject);
+    request_es_no_body("del", `/${index}`).then(() => resolve(index)).catch(reject);
   });
 }
 
 function delete_indices(indices) {
-  return Promise.all(_.map(indices, (index) => delete_index(index)));
+  return Promise.all(_.map(indices, delete_index));
 }
 
 export function clear_old_indices() {
   return get_indices().then(filter_old_indices).then(delete_indices);
 }
+
+function get_aliases() {
+  return new Promise((resolve, reject) => {
```

Comment:
This is my first time using promises, the more I learn the more I like them: https://github.com/Clever/elasticsearch-toolbox/pull/4/commits/6792ab48afe00a85a94f6177b0d7f0ad728c2243